Assets/Scripts/Framework/GameLoop/LoseScreenController.cs

```diff
@@ -0,0 +1,24 @@
+using UnityEngine.SceneManagement;
+using UnityEngine;
+using UnityEngine.Events;
+
+public sealed class LoseScreenController : MenuController
+{
+    [SerializeField] private GameObject activateLoseScreen;
```

Comment:
```suggestion
    [SerializeField] private GameObject loseScreen;
```
Ik vraag me af waarom je "activate" er bij zet. Het is gewoon een UI screen. Je het wel uit of aan, maar dan hoort de naam ook dat te reflecteren. Helaas kan je varible in runtime niet veranderen. Ik zo het gewoon weg laten.